pkg/user/manager/nextcloud/nextcloud.go

```diff
@@ -106,29 +106,30 @@ func (um *Manager) SetHTTPClient(c *http.Client) {
 func getUser(ctx context.Context) (*userpb.User, error) {
 	u, ok := ctxpkg.ContextGetUser(ctx)
 	if !ok {
-		err := errors.Wrap(errtypes.UserRequired(""), "nextcloud storage driver: error getting user from ctx")
+		err := errors.Wrap(errtypes.UserRequired(""), "nextcloud user manager: error getting user from ctx")
 		return nil, err
 	}
 	return u, nil
 }
 
 func (um *Manager) do(ctx context.Context, a Action, username string) (int, []byte, error) {
```

Comment:
this function seems to be similar to the one used for the storage driver, maybe extract it somewhere?